fe/fe-core/src/main/java/com/starrocks/external/iceberg/IcebergUtil.java

```diff
@@ -0,0 +1,108 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.external.iceberg;
+
+import com.starrocks.catalog.IcebergTable;
+import com.starrocks.external.hive.HdfsFileFormat;
+import org.apache.iceberg.FileFormat;
+import org.apache.iceberg.Snapshot;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.TableScan;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.expressions.UnboundPredicate;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class IcebergUtil {
+    /**
+     * Get Iceberg table identifier by table property
+     */
+    public static TableIdentifier getIcebergTableIdentifier(IcebergTable table) {
+        return TableIdentifier.of(table.getDb(), table.getTable());
+    }
+
+    /**
+     * Get Iceberg table identifier by table property
+     */
+    public static TableIdentifier getIcebergTableIdentifier(String db, String table) {
+        return TableIdentifier.of(db, table);
+    }
+
+    /**
+     * Returns the corresponding catalog implementation.
+     */
+    public static IcebergCatalog getIcebergCatalog(IcebergTable table)
+            throws StarRocksIcebergException {
+        IcebergCatalogType catalogType = table.getCatalogType();
+        return getIcebergCatalog(catalogType, table.getIcebergHiveMetastoreUris());
+    }
+
+    /**
+     * Returns the corresponding catalog implementation.
+     */
+    public static IcebergCatalog getIcebergCatalog(IcebergCatalogType catalogType, String metastoreUris)
+            throws StarRocksIcebergException {
+        switch (catalogType) {
+            case HIVE_CATALOG: return IcebergHiveCatalog.getInstance(metastoreUris);
+            default:
+                throw new StarRocksIcebergException(
+                        "Unexpected catalog type: " + catalogType.toString());
+        }
+    }
+
+    /**
+     * Get hdfs file format in StarRocks use iceberg file format.
+     * @param format
+     * @return HdfsFileFormat
+     */
+    public static HdfsFileFormat getHdfsFileFormat(FileFormat format) {
+        switch (format) {
+            case ORC:
+                return HdfsFileFormat.ORC;
+            case PARQUET:
+                return HdfsFileFormat.PARQUET;
```

Comment:
Will this mean we will use the `org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat`  to read the parquet row-group, right ?

Will the row-group max-min filter push down also effect the read process (which is similar to the iceberg parquet reader , https://github.com/apache/iceberg/blob/827b6c86108eec7b6de25f61022c7f8b5dda481c/parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java#L107) ?

Reply:
we will read parquet by https://github.com/StarRocks/starrocks/blob/main/be/src/exec/parquet_reader.cpp 
The format here will send to be through thrift which will be used to init hdfs file scanner
```
HdfsScanner* scanner = nullptr;
    if (hdfs_file_desc.hdfs_file_format == THdfsFileFormat::PARQUET) {
        scanner = _pool->add(new HdfsParquetScanner());
    } else if (hdfs_file_desc.hdfs_file_format == THdfsFileFormat::ORC) {
        scanner = _pool->add(new HdfsOrcScanner());
    } else {
        std::string msg = fmt::format("unsupported hdfs file format: {}", hdfs_file_desc.hdfs_file_format);
        LOG(WARNING) << msg;
        return Status::NotSupported(msg);
    }
```